packages/app/package.json

```diff
@@ -57,7 +57,7 @@
     "react-native": "*"
   },
   "dependencies": {
-    "firebase": "11.10.0"
+    "firebase": "12.0.0"
```

Comment:
you're kidding, it can't be that easy can it ?

Reply:
I made it breaking just to alert users really, it will be part of breaking change release with DL removal + android/iOS SDK major version updates. But now that you mention those two min requirements, I'll add them to migration guide 👍